runtime/development/src/lib.rs

```diff
@@ -2073,15 +2124,16 @@ construct_runtime!(
 		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 120,
 		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Config<T>, Event<T>, Origin} = 121,
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 122,
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 123,
+		// Removed: DmpQueue = 123
```

Comment:
We will need to migrate any existing messages from this queue to the new once, right?

Reply:
Great catch! The DMP Queue has a lazy migration via `on_idle`. So we need to keep it in the runtime @lemunozm 

> The undeployment of the pallet commences through the on_idle hook step-by step and moves all the remaining messages into the DmpSink.
> 
> ```
> impl cumulus_pallet_dmp_queue::Config for Runtime {
>	type DmpSink = MessageQueue;
>	...
> ```
> The pallet can be removed after the lazy migration is done and event Completed is emitted.

Source: https://github.com/paritytech/polkadot-sdk/pull/1246

Reply:
Done [b5a7d89](https://github.com/centrifuge/centrifuge-chain/pull/1833/commits/b5a7d895819d89bcd8d66d203fb648e388378cc3)